jde/modules/ROOT/pages/demo_poll_mbf_events.adoc

```diff
@@ -0,0 +1,332 @@
+
+After reading this guide, you should be able to poll for transaction (Master Business Function) events created on the server.
+
+At design time, the _PortX JDE Connector_ allows you to discover functions by name providing full access to all Business Function signatures. Given that discovery is a heavy duty process, the Connector manages a cache repository for all function’s metadata used at least once.
+
+The results of a function call will be injected into the flow as a Map containing key-value pairs with the invocation’s output values. This approach gives the end user freedom of choice to construct complex flows and manage custom exception strategies.
+
+Furthermore, you can have all functions in the flow, processing within the same transaction, and commit or rollback all changes.
+
+=== Prerequisites
+
+This document assumes that you are familiar with Oracle’s _JD Edwards EnterpriseOne™_ basics, Mule the https://docs.mulesoft.com/anypoint-studio/v/6/download-and-launch-anypoint-studio[Anypoint™ Studio] interface.
+
+=== Requirements
+
+*Review the* link:/[PortX JDE Connector for Mule 4 – Getting Started Guide] for a detailed list of requirements and setup prerequisites.
+
+JDE outbound transactions requires that you set a _*Processing Option*_ specifying a transaction type. Additionally, some entry programs enable you to specify a version of the Master Business Function Processing Options program that, in turn, enables you to specify a version of the Interoperability Processing Options program.
+
+For example, to get customer entered by the application _*Customer Master Information*_ (P03013) it is necessary to setup it’s processing options to enable interoperability. To do this you will need to log into JDE, and change the processing options for this application. Firstly, we need to determine which version of the Master Business Function is being called by the application. To do this, we will need to review the processing options.
+
+image:image1_demo_poll_mbf_events.png[image,width=601,height=332]
+
+image:image2_demo_poll_mbf_events.png[image,width=601,height=347]
+
+The Customer Master Business Function version that is used for the version of the application is ZJDE0002, so we need to now review the Customer Master MBF (P0100042) processing options.
+
+image:image3_demo_poll_mbf_events.png[image,width=601,height=314]
+
+image:image4_demo_poll_mbf_events.png[image,width=601,height=381]
+
+If the Transaction Type is not yet set, we need to set it to the appropriate value.
+
+For more on setting transaction types, please refer to the _Oracle JD Edwards EnterpriseOne_ documentation
+
+== *Demo – Poll Transaction (MBF) Events*
+
+*NOTE*: It is recommended that you update AnyPoint Studio before starting with a _PortX JDE Connector_ project.
```

Comment:
Asciidoc has special formatting for what it calls "admonitions." There's an explanation at https://asciidoctor.org/docs/user-manual/#admonition but, basically, for a note, all you need is 

NOTE: <text of note>

So here all you need to do it lose the asterisks.